Architecture/TodoApp/android/core/src/main/java/com/novoda/todoapp/tasks/data/source/InMemoryLocalTaskDataSource.java

```diff
@@ -22,12 +22,12 @@
     private Tasks localTasks = Tasks.asSynced(
             ImmutableList.<Task>of(
                     Task.builder()
-                        .id(Id.from("42"))
```

Comment:
It was just moved down, it is a switch since now items are ordered I found it easier to see them in the same order here. Not strictly needed.